RELEASE_NOTES.md

```diff
@@ -2,9 +2,43 @@
 
 While [CHANGELOG.md](./CHANGELOG.md) contains detailed documentation and links to all the source code changes in a given release, this document is intended to be aimed at a more comprehensible version of the contents of the release from the point of view of users of Refinery.
 
+## Version 2.7.0
+
+This release is a transitional release, laying the groundwork for substantial future changes to Refinery.
+
+### Publish/Subscribe on Redis
+In this release, Redis is no longer a database for storing a list of peers.
+Instead, it is used as a more general publish/subscribe framework for rapidly sharing information between nodes in the cluster.
+Things that are shared with this connection are:
+
+- Peer membership
+- Stress levels
+- News of configuration changes
+
+Because of this mechanism, Refinery will now react more quickly to changes in any of these factors.
+In particular, all of the nodes in the cluster will now go into and out of stress relief together.
+Also, when one node detects a configuration change, all of its peers will be told about it immediately.
```

Comment:
Do we want to add a disclaimer about the delay introduced in k8s if configurations are mounted as a ConfigMap?